modules/gdscript/language_server/gdscript_language_server.cpp

```diff
@@ -36,6 +36,8 @@
 #include "editor/editor_node.h"
 
 GDScriptLanguageServer::GDScriptLanguageServer() {
+
+	_EDITOR_DEF("network/language_server/remote_host", host);
```

Comment:
Remove blank line at the beginning of the function:

```suggestion
	_EDITOR_DEF("network/language_server/remote_host", host);
```